R/inst/tutorials/04-helloaws/README.md

```diff
@@ -15,10 +15,12 @@ use the client to access information about any flow from anywhere.**
 1. Configure your sandbox: https://docs.metaflow.org/metaflow-on-aws/metaflow-sandbox
 
 #### To play this episode:
-1. ```cd metaflow-tutorials/R```
-2. ```Rscript 04-helloaws/helloaws.R run```
+1. ```cd tutorials/04-helloaws```
+2. ```Rscript helloaws.R run```
 3. Open ```helloaws.Rmd' in your local RStudio 
```

Comment:
Do we need `RStudio` here?

Reply:
Yes RStudio is mostly for running the R markdown files